samigo/samigo-app/src/webapp/css/timerbar.css

```diff
@@ -0,0 +1,117 @@
+.ui-progressbar {
+	position: relative;
+	background: #A0A0A0;
+	margin-bottom: 15px;
+}
+
+.progress-label {
+	position: absolute;
+	width: 100%;
+	top: 3px;
+	font-weight: bold;
+	text-shadow: 1px 1px 0 #fff;
+	text-align: center;
+}
+
+#dialog-timeout {
+	display: none;
+}
+
+#timeoutWarning {
+	display: none;
+	background-color: #f2dede;
+	border-color: #ebccd1;
+	color: #a94442;
+	padding: 15px;
+	margin-bottom: 20px;
+	border: 1px solid transparent;
+	border-radius: 4px;
+	margin: 10px 15px 0px 15px;
+}
+
+#progressbar {
+	width: 98%;
+	margin: auto;
+	margin-top: 10px;
+}
+
+#timerBlock {
+	position: absolute;
+	top: 0px;
+	width: 75%;
+	background-color: #000036; 
+	margin: 0;
+	margin-left: 15px;
+	padding: 0;
+	z-index: 2;
+}
+
+#timerBlank {
+	height: 110px;
+	border: 0;
+	padding: 0;
+	margin: 0;
+}
+
+#showHide {
+	text-align: center;
+	margin-top: 8px;
+	margin-bottom: 5px;
+	cursor: pointer;
+	outline: none;
```

Comment:
is this element focusable?

Reply:
This is fixed in my last push, I also added keyboard accessibility to the "Question Progress" popout that was not there before.

Reply:
I only see one commit ....

Reply:
all squashed

Reply:
The block below was added. div#showHide:focus to display an outline on keyboard navigation.

Reply:
I'm still not comfortable with this.... have you run it past an accessibility expert?

Reply:
I can have Tiffany look at it. but what exactly, it's ability to be selected via keyboard, or the styling of the :focus itself?

Reply:
I'm concerned with focus manipulation and would rather it be left to default behaviors instead of being overridden here. Individual outline and focus manipulation just raises a big red flag for me.

Reply:
Ohhh, well in this case, the default is nothing, since it object is a div, this isn't overriding a default, it simply providing something at all to indicate the object is focused

Reply:
Tiffany looked at it, only suggested making the outline 3px instead of 2px, that is updated and squashed into this commit. Screen cap and such on JIRA https://jira.sakaiproject.org/browse/SAK-40489

Reply:
What's the logic in overriding default outline in each tool / component / etc? I just don't get why it's a part of this.

Reply:
There is no default outline for this element, I removed all the outline overrides for the other elements, but needed to add an outline here as by default there was no indicator at all.

Reply:
Either way, I removed the outline: none there, since in either case it's unnecessary.

Do we prefer to use the difficult to see default, or leave in the new 3px cleanly padded?

Reply:
I'm obviously super sensitive about anything a11y related as regressions here cost us so much $$ and time. Removing any mention of outline in your new CSS would be appreciated. I'm happy for @fostersdesign and you and others to talk about a more attractive but accessible outline for all Sakai focusable elements!

Reply:
okay, done. I added a screen cap to the issue in JIRA just for comparison, and maybe conversation on better sakai defaults for div keyboard selection.